docs/pages/management/diagnostics/tracing.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Distributed Tracing
+description: How to enable tracing within Teleport.
+---
+
+Teleport leverages [OpenTelemetry](https://opentelemetry.io/) to generate traces
+and export them to any [OpenTelemetry Protocol (OTLP)](https://opentelemetry.io/docs/reference/specification/protocol/otlp/)
+capable exporter. In the event that your telemetry backend doesn't support receiving OTLP traces, you may be able to
+leverage the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) to proxy traces from OTLP
+to a format that your telemetry backend accepts.
+
+## Configure Teleport
+
+In order to enable tracing in Teleport, add the following section in `teleport.yaml`. For a detailed description of
+these configuration fields, see the [configuration reference](./reference/configuration.mdx) page.
+
+<Details title="Tracing Service Configuration" min="8.3.18,9.3.15,10.1">
+```yaml
+tracing_service:
+   enabled: yes
+   exporter_url: grpc://collector.example.com:4317
+   sampling_rate_per_million: 1000000
+```
+</Details>
+
+It is important to choose the sampling rate wisely. Sampling at a rate of 100% could have a negative impact on the
+performance of your cluster. Teleport honors the parent span's sampling rate, which means that even when the
+`tracing_service` is enabled and the sampling rate is 0, if Teleport receives a request that has a span which is
+sampled, then Teleport will sample and exports all spans that are generated in response to that request.
+
+After updating `teleport.yaml`, start Teleport as usual using `teleport start`.
```

Comment:
```suggestion
After updating `teleport.yaml`, start your `teleport` instance to apply the new configuration.
```
Suggesting a vaguer phrasing since users who installed Teleport via package manager on systemd-enabled systems would be using systemctl.